client/data/plugins/use-scheduled-updates-notification-settings-mutation.ts

```diff
@@ -0,0 +1,59 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useCallback } from 'react';
+import wpcomRequest from 'wpcom-proxy-request';
+import type { ScheduledUpdatesNotificationSettings } from './use-scheduled-updates-notification-settings-query';
+import type { SiteSlug } from 'calypso/types';
+
+export type CreateRequestParams = {
+	success: boolean;
+	failure: boolean;
+};
+
+export function useScheduledUpdatesNotificationSettingsMutation(
+	siteSlug: SiteSlug,
+	queryOptions = {}
+) {
+	const queryClient = useQueryClient();
+
+	const mutation = useMutation( {
+		mutationKey: [ 'create-update-schedule', siteSlug ],
```

Comment:
Should we rename the mutation key here to be more specific for the notification context?

Reply:
Oops, I clearly overlooked this one.